core-bundle/contao/dca/tl_content.php

```diff
@@ -1047,6 +1057,20 @@ public function adjustDcaByType($dc)
 		{
 			$GLOBALS['TL_DCA']['tl_content']['config']['switchToEdit'] = true;
 		}
+
+		if ($objCte->ptable == 'tl_content')
+		{
+			$objParent = ContentModel::findByPk($objCte->pid);
+
+			// Dynamically add the section headline to accordion child elements
+			if ($objParent && $objParent->type == 'accordion')
+			{
+				PaletteManipulator::create()
+					->addLegend('section_legend', 'type_legend')
+					->addField('sectionHeadline', 'section_legend', PaletteManipulator::POSITION_APPEND)
+					->applyToPalette($objCte->type, 'tl_content');
+			}
+		}
```

Comment:
This might be a use case for the `onpalette_callback` if we want to merge https://github.com/contao/contao/pull/6638

Reply:
It definitely is, because this code does not work in "edit multiple" mode. 🤦‍♂️

Reply:
Fixed in a584e75b736cf06e16894395e0516c4875cf36b8.